pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -422,8 +421,28 @@ func isEmptyChecksum(v string) bool {
 		"da39a3ee5e6b4b0d3255bfef95601890afd80709": true,
 		// sha256 empty file
 		"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855": true,
+		// sha224 empty file
+		"d14a028c2a3a2bc9476102bb288234c415a2b01f828ea62ac5b3e42f": true,
+		// sha384 empty file
+		"38b060a751ac96384cd9327eb1b1e36a21fdb71114be07434c0cc7bf63f6e1da274edebfe76f65fbd51ad2f14898b95b": true,
 		// sha512 empty file
 		"cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e": true,
+		// MD5 empty file
```

Comment:
seems like we are missing a few per the issue: [BLAKE3](https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE3), [MD2](https://tools.ietf.org/html/rfc1319), [MD4](https://tools.ietf.org/html/rfc1320), [MD6](https://groups.csail.mit.edu/cis/md6/)?

Reply:
I want to address those in a subsequent PR.

There is only so much I can copy paste the tests and not make mistakes ;)